docs/en/dataobjects.md

```diff
@@ -74,11 +76,29 @@ The following will construct such a list, ensuring that only objects valid in th
 locale are given.
 
 ```php
-public function getCMSFields() {
+public function getCMSFields()
+{
 	$fields = parent::getCMSFields();
 	// Causes filtering to be enabled within the admin
 	$banners = $this->Banners()->setDataQueryParam(FluentFilteredExtension::FILTER_ADMIN, true);
 	$fields->addFieldsToTab('Root.Banners', new CheckboxSetField('Banners', 'Banners', $banners));
 	return $fields;
 }
 ```
+
+### Extensions
+
+As mentioned in the CMS Fields heading, it is necessary to ensure that the `updateCMSFields` extension method is called once (and only once) on each `FieldList` object. This means that if you're applying the `FluentExtension` to a `DataObject`, other extensions that manipulate the `FieldList` may have problems.
```

Comment:
I would clarify `the `FieldList` may have problems.` here.

```
Fields added by other extensions will not have the necessary decorations applied to it by the FluentExtension, unless it happens to be applied before FluentExtension. Since extension order cannot be guaranteed, this poses a problem.
```